src/util/get-object-value.js

```diff
@@ -11,5 +13,5 @@ module.exports = function (object, path, defaultValue) {
     return (a || {})[b];
   }, object);
 
-  return value || defaultValue;
+  return _.isUndefined(value) ? value : defaultValue;
```

Comment:
Hmmm wouldn't it be `return _.isUndefined(value) ? defaultValue : value;`

Reply:
😁